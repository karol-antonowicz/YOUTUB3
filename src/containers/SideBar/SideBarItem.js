import React from 'react';
import {Icon, Menu} from 'semantic-ui-react';
import './SideBarItem.scss'

export const SideBarItem = props => {

    return (
        <div>

        </div>
    )
}