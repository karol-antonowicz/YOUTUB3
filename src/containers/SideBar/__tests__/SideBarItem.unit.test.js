import React from 'react';
import {shallow} from 'enzyme';
import {SideBarItem} from '../SideBarItem';

test('renders SideBarItem', ()=>{
    const wrapper = shallow(
        <SideBarItem/>
    );
    expect(wrapper).toMatchSnapshot();
});