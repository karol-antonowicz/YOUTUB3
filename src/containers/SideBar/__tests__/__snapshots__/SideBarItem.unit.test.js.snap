// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBarItem 1`] = `
<MenuItem
  className="side-bar-item "
>
  <div
    className="sidebar-item-alignment-container"
  >
    <span>
      <Icon
        as="i"
        size="large"
      />
    </span>
    <span />
  </div>
</MenuItem>
`;
